client/docs/issues/settings-ui-redesign.md

```
@@ -0,0 +1,35 @@
+## 제목
+✨ [기능개선][UI/UX] 설정 화면 UI 개선 — 콘텐츠 보강 및 시각적 완성도 향상
+
+## 본문
+
+📝 현재 문제점
```

markdownlint-cli2 (0.22.0)

[warning] Heading style
Expected: atx; Actual: setext

(MD003, heading-style)